packages/tokamak/contracts-bedrock/src/libraries/Address.sol

```diff
@@ -0,0 +1,222 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I'm sorry. I missed these files.
How about changing the directory structure? I'm sorry again my fault,,, 😢😢

src
  tokamak-contracts
    USDC
      L1
      L2
      libraries
      universal

Reply:
@rlgns98kr 
In L2, is there no need to separate the usdc contract and usdcBridge contract files??

Reply:
Oh, I'm sorry for my late.
About sub directory, I think yours is greeat! So I think you can keep your seperate structure in the L2!